lib/pghero/methods/queries.rb

```diff
@@ -33,7 +33,7 @@ def long_running_queries
       # from https://wiki.postgresql.org/wiki/Lock_Monitoring
       # and https://big-elephants.com/2013-09/exploring-query-locks-in-postgres/
       def blocked_queries
-        select_all <<-SQL
+        select_all_with_data_protection <<-SQL
```

Comment:
There are two query columns here: `blocked_query` and `current_or_recent_query_in_blocking_process`

Reply:
thanks, I really missed it.